homework/index.js

```diff
@@ -29,18 +14,94 @@
     });
     return elem;
   }
+  function createRepositoryDescription(repoContainer, repository) {
+    repoContainer.innerHTML = '';
+    const table = createAndAppend('table', repoContainer);
+    const tBody = createAndAppend('tbody', table);
+    const details = ['Repository:', 'Description:', 'Forks:', 'Updated:'];
+    details.forEach(detail => {
+      const tr = createAndAppend('tr', tBody);
+      createAndAppend('td', tr, { class: 'label', text: detail });
+      createAndAppend('td', tr, { id: detail });
+    });
+    const secondTd = document.getElementById('Repository:');
+    const link = createAndAppend('a', secondTd, {
+      href: repository.html_url,
+      target: '_blank',
+    });
+    link.innerText = repository.name;
+    document.getElementById('Description:').innerText = repository.description;
+    document.getElementById('Forks:').innerText = repository.forks;
+    document.getElementById('Updated:').innerText = new Date(
```

Comment:
There is no best way of doing code, but you may need to change the `id` names to be without `:` so if you used it into your Css it won't cause any problems.